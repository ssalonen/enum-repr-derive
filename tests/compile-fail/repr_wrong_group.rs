extern crate enum_repr_derive;
use enum_repr_derive::{Into, TryFrom};

#[repr[i8]]
#[derive(TryFrom, Into, Copy, Clone, Debug, PartialEq)]
enum Foo {
    VAR1 = -1,
    VAR2 = -3,
}

fn main() {}
