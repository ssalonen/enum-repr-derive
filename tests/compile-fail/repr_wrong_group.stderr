error: Repr is malformed, expecting repr(TYPE)
 --> $DIR/repr_wrong_group.rs:4:7
  |
4 | #[repr[i8]]
  |       ^^^^

error: expected one of `(` or `=`, found `[`
 --> $DIR/repr_wrong_group.rs:4:7
  |
4 | #[repr[i8]]
  |       ^ expected one of `(` or `=` here
